ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py

```diff
@@ -260,6 +260,28 @@ def test_jax_special_rpow(
     )
 
 
+# __all__
+@handle_frontend_test(
+    fn_tree="numpy.ndarray",  # dummy fn_tree
```

Comment:
This should refer to the function you're implementing instead of `numpy.ndarray`. Refer to the implementations of other `jax devicearray` functions to get an idea. 

Also, I see that the commit still contains a file from the `.idea` folder. You can check online on how to add only specific files to a commit. Maybe [this link](https://www.nobledesktop.com/learn/git/stage-commit-files) would be helpful.